src/libraries/System.Private.CoreLib/src/System/DateTime.cs

```diff
@@ -1405,6 +1405,7 @@ internal void GetDate(out int year, out int month, out int day)
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         internal void GetTime(out int hour, out int minute, out int second)
         {
+            // TODO: https://github.com/dotnet/runtime/issues/5213
```

Comment:
I'd honestly prefer these to instead just switch to using `ulong.DivRem`, that should produce the same code today and will automatically be updated in the future when any JIT optimization gets added.

Reply:
I switched to DivRem except for in Decimal.DecCalc, where we the TODOs in this PR and the ones that already exists are heterogenous.